### PR TITLE
Simplify merging to different branches

### PR DESCRIPTION
**Pull request by kargakis**
@tdawson this will help in not requiring to update the script every time we branch out to a new release (apart from updating `OSE_MASTER` and `OSE_MASTER_BRANCHED`).

### CODE REVIEW

File: scripts/merge-and-build.sh [hunk 1]
```diff
@@ -143,28 +143,28 @@ cd ${WORKPATH}
...
+  if [[ "${BUILD_MODE}" == "online/stg" ]]; then
+    CURRENT_BRANCH="stage"
+    UPSTREAM_BRANCH="upstream/stage"
+  else if [[ "${OSE_VERSION}" == "${OSE_MASTER}" ]]; then
+    CURRENT_BRANCH="master"
+    UPSTREAM_BRANCH="upstream/master"
+  else if [[ "${OSE_VERSION}" == "${OSE_MASTER_BRANCHED}" ]]; then
+    CURRENT_BRANCH="enterprise-${OSE_VERSION}"
+    UPSTREAM_BRANCH="upstream/release-1.${MINOR}"
+  fi
```
Comment (tdawson) [inline scripts/merge-and-build.sh]: Now that 3.5 is released, that should be
`UPSTREAM_BRANCH="upstream/release-${MAJOR}.${MINOR}"`

Comment (kargakis) [inline scripts/merge-and-build.sh]: Updated to `upstream/release-${OSE_VERSION}` then

**Comment by kargakis** [comment]
@tdawson ptal

**Comment by tdawson** [comment]
👍 
I like it.  It is much cleaner and easier to ready.

**Pull request merged**